packages/multichain-network-controller/src/MultichainNetworkController.test.ts

```diff
@@ -0,0 +1,155 @@
+import { ControllerMessenger } from '@metamask/base-controller';
+
+import type {
+  MultichainNetworkStateControllerActions,
+  MultichainNetworkControllerEvents,
+} from './MultichainNetworkController';
+import { bitcoinCaip2ChainId } from './constants';
+import { MultichainNetworkController } from './MultichainNetworkController';
+
+const name = 'MultichainNetworkController';
+
+type AllowedActions = MultichainNetworkStateControllerActions | {
```

Comment:
What are your thoughts on importing this and `AllowedEvents` from `./MultichainNetworkController`?